app/views/wiki/edit.html.erb

```diff
@@ -75,6 +75,7 @@
  
       <a id="publish" tabindex="5" class="publish btn btn-primary btn-lg"><%= t('wiki.edit.publish') %></a>
       <a tabindex="6" data-previewing-text="<%= t('wiki.edit.previewing_text') %>" class="btn btn-default btn-lg preview-btn" onClick="$E.toggle_preview()"><%= t('wiki.edit.preview') %></a>
+      <a class="btn btn-danger btn-lg" href="/wiki?id=#{id}">Cancel</a>
```

Comment:
Awesome.Correct class for cancel button, just replacing href will get it to work.Thanks.